deploy-help/delete-database.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash -e
+
+cluster_name="$1"
+deployment_dir=~/deployment
+kubernetes_build_dir="$deployment_dir/concent-deployment/kubernetes/build"
+
+function run_kubernetes_script {
+    executed_script="$1"
+    cd $kubernetes_build_dir/
+    ./$executed_script
+}
+
+if [ -z "$cluster_name" ]; then
+    echo 'Please specify the name of database'
+    exit 1
+fi
+
+# Confirm executing script
+read -p "Warning must execute after deploy.sh script. Are you sure about execute this script? It's destroy database[y/n]" -n 1 -r
```

Comment:
Please reword:

``` bash
echo "This script won't work correctly unless you run `deploy.sh` first because it depends on containers built by it."
read -p "This script removes everything from the databases on the dev cluster. Are you sure you want to continue?[y/N]" -n 1 -r
```

Reply:
Actually, when you merge this script with `deploy.sh`, this question will not be necessary.